### ISSUE TITLE
Can't run the AppImage build on Debian 12

### ISSUE THREAD

**Issue by Jeropeesee-Bashan** [Vinegret43/dispute#1]
The AppImage build for Linux that's from the releases page won't run on Debian 12. Running the executable results in:
```
dispute: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by dispute)
```

As Debian 12 uses glibc 2.36, maybe it'd be a good idea to somehow soften the requirements to glibc 2.36 or lower.

**Comment by Vinegret43** [comment]
I actually knew about this issue and specifically built Dispute AppImage inside a Debian 11 docker container, or at least I thought so. Seems like I ended up uploading an AppImage assembled on my host OS instead. There should be a new release of Dispute pretty soon which should fix this, though

**Comment by Vinegret43** [comment]
@Jeropeesee-Bashan, please report whether this issue is fixed for you in the [1.1 Dispute release](https://github.com/Vinegret43/dispute/releases/tag/1.1.0)

**Comment by Jeropeesee-Bashan** [comment]
Yep, works just as expected! 👍
